test/templates/layout.njk

```diff
@@ -7,23 +7,38 @@
     <title>{{ id }}</title>
     <link rel="stylesheet" href="static/layout.css">
     <style>
-        {% if ie11 %}
-            {% set sass = 'fundamental-styles-ie11.scss' %}
-        {% elif selfContained %}
-            {% if id === 'icon' %}
-                {% set sass = 'icons.scss' %}
-            {% elif id === 'radio' or id === 'checkbox' or id === 'input' %}
-                {% set sass = 'components/form.scss' %}
-            {% elif id === 'panel' or id === 'panel-grid' or id === 'layout' %}
-                {% set sass = 'layout.scss' %}
+        {% if selfContained %}
+            {% set sass = id + '.scss'  %}
+            {{ sass | component_css }}
```

Comment:
All the self-contained component pages will now be missing styles except for button, but I think that is what we want so we can validate them as more and more components are added to the self-contained styles.